### PR TITLE
Complete readme with usages

### PR DESCRIPTION
**Pull request by JulienMattiussi**
Complete readme with usages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+
+## Installation
+
+`make install`
+
+## Usage
+
+`NAME="[name_of_the_run]" docker-compose up`
+
+See the results displayed on :
```
Comment (alanpoulain) [inline README.md:18]:
```suggestion
See the results displayed on:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
 # ARGOS
+
 Auditing Reporter for Greener Ops Strategies
+
+## Requirements
+
+- Docker
+- Docker compose
```
Comment (alanpoulain) [inline README.md:8]:
```suggestion
- Docker Compose
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,20 @@
...
+
+## Installation
+
+`make install`
+
+## Usage
+
+`NAME="[name_of_the_run]" docker-compose up`
```
Comment (alanpoulain) [inline README.md:16]: Would be great if it was a make command.

Comment (JulienMattiussi) [reply README.md:16]: You're right. It might be done after the server refactor @ThieryMichel  is working on because if I do it now, it will be erased.

**Pull request merged**